HOW_TO_HANDLE_PULL_REQUESTS.md

```diff
@@ -0,0 +1,51 @@
+# How to Handle Pull Requests
+
+This document is for whoever has the ability to merge pull requests in the Git repositories associated with BigchainDB.
+
+If the pull request is from an employee of ascribe GmbH, then you can ignore this document.
+
+If the pull request is from someone who is _not_ an employee of ascribe, then:
+
+* Have they agreed to the Individual Contributor Agreement in the past? (Troy, Greg, and others have a list.) If yes, then you can merge the PR and ignore the rest of this document.
```

Comment:
This document is mainly for ascribe employees and I think our first names are more clear than our github handles. For example, one might have to think a bit, or dig around, to find out who @ttmc is.